referenceframe/frame_system.go

```diff
@@ -150,12 +150,12 @@ func NewFrameSystem(name string, parts []*FrameSystemPart, additionalTransforms
 	return fs, nil
 }
 
-// World returns the base world referenceframe.
```

Comment:
why are you deleting the word referenceframe from the comments?

Reply:
Oof bad find and replace 🤦

Reply:
I guess this gives me a reason to go back and clean up these comments